modules/install-secured-cluster-operator.adoc

```diff
@@ -14,6 +14,12 @@ endif::[]
 [role="_abstract"]
 You can install Secured Cluster services on your clusters by using the Operator, which creates the `SecuredCluster` custom resource. You must install the Secured Cluster services on every cluster in your environment that you want to monitor.
 
+[IMPORTANT]
+====
+* When you install {product-title} for the first time, you must first install the `Central` custom resource because the `SecuredCluster` custom resource installation is dependent on certificates that Central generates.
+* Red{nbsp}Hat recommends that you install the {product-title} `SecuredCluster` custom resource in its own project and not in the project in which you have installed the {product-title} Operator. You must not install it in projects whose name start with `kube`, `openshift`, `redhat` nor in project `istio-system`.
```

Comment:
For reducing wording and ease of translation, maybe rewrite as: "...custom resource in its own project and not in the project where you installed the {product-title} Operator. "